onnx/defs/math/defs.cc

```diff
@@ -1168,4 +1168,33 @@ ONNX_OPERATOR_SET_SCHEMA(
             {"tensor(float16)", "tensor(float)", "tensor(double)"},
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
+
+
+static const char* Sign_ver9_doc = R"DOC(
+lalala
```

Comment:
Documentation? The documentation needs to be generated also.

Reply:
lol yeah, I havent delete all debugging info either, just wanna ask for you and @linkerzhang 's opinion on this Op.